samples/app-dev/Flow-Design-Concepts/allOf_oneOf-anyOf_construct_schema/OpenAPISpecs/AllOfKeywordOAS30GetPost.json

```diff
@@ -0,0 +1,357 @@
+{
+  "openapi": "3.0.1",
+  "info": {
+    "title": "FleetOps.LeaseObject.Api 0.3",
+    "description": "FleetOps.LeaseObject.Api - FLOGO-8896",
+    "version": "2.0.2"
+  },
+  "servers": [
+    {
+      "url": "https://c3f5f68230e5f1df8a0bf25726-integration.local.cic2apps.pro:443/udpr3vpe6xs5a5fr7gafb6g6bl4ujwrl",
+      "description": "dev"
+    },
+    {
+      "description": "tst",
+      "url": "https://eu-tst-api.leaseplan.com"
```

Comment:
Please edit this URL to something else, we can't have leaseplan in the spec
"https://c3f5f68230e5f1df8a0bf25726-integration.local.cic2apps.pro:443/wewe8728323rwrwe"